File: Latihan 1.2/Java/Sepakbola.java
public class Sepakbola
{	
	/*Nama Atribut*/
	private String nama;	
	private	String negara;
	private	int tahun;
	private	int pemain;

	public Sepakbola(){}
	
	public Sepakbola(String nama, String negara, int tahun, int pemain){
		this.nama = nama;
		this.negara = negara;
		this.tahun = tahun;
		this.pemain = pemain;	
	}

	public void setNama(String nama){
		this.nama = nama;
	}

	public String getNama(){
		return nama;
	}

	public void setNegara(String negara){
		this.negara = negara;
	}

	public String getNegara(){
		return negara;
	}

	public void setTahun(int tahun){
		this.tahun = tahun;
	}

	public int getTahun(){
		return tahun;
	}

	public void setPemain(int pemain){
		this.pemain = pemain;
	}

	public int getPemain(){
		return pemain;
	}
	
};

File: Latihan 1.2/Cpp/Sepakbola.cpp
#include <iostream>
#include <string>

using namespace std;

class Sepakbola
{	
	/*Nama Atribut*/
	private:
		string nama;
		string negara;
		int tahun;
		int pemain;

	public:
		Sepakbola()
		{
			this->nama = "";
			this->negara = "";
			this->tahun = 0;
			this->pemain = 0;	
		}

		Sepakbola(string nama, string negara, int tahun, int pemain)
		{
			this->nama = nama;
			this->negara = negara;
			this->tahun = tahun;
			this->pemain = pemain;		
		}

	void setNama(string nama){
		this->nama = nama;
	}

	string getNama(){
		return this->nama;
	}

	void setNegara(string negara){
		this->negara = negara;
	}

	string getNegara(){
		return this->negara;
	}

	void setTahun(int tahun){
		this->tahun = tahun;
	}

	int getTahun(){
		return this->tahun;
	}

	void setPemain(int pemain){
		this->pemain = pemain;
	}

	int getPemain(){
		return this->pemain;
	}

	// destructor
	~Sepakbola(){}
	
};